eng/common/scripts/Invoke-GitHubAPI.ps1

```diff
@@ -90,13 +90,29 @@ function Invoke-GitHubAPIGet {
   return $resp
 }
 
-function SplitMembers ($membersString)
+function Split-GitHubAPIParameterMembers ($membersString)
 {
   if (!$membersString) { return $null }
   return @($membersString.Split(",") | % { $_.Trim() } | ? { return $_ })
 }
 
-function List-PullRequests {
+function Set-GitHubAPIParameters ($members,  $parameterName, $parameters, $allowEmptyMembers=$false)
```

Comment:
Instead of doing this extra work maybe we should just make the split function handle it.